core/tests/protractor_desktop/extensions.js

```diff
@@ -101,8 +101,11 @@ describe('rich-text components', function() {
       // TODO(pranavsid98): This error is caused by the upgrade from Chrome 60
       // to Chrome 61. Chrome version at time of recording this is 61.0.3163.
       'chrome-extension://invalid/ - Failed to load resource: net::ERR_FAILED',
-      'The target origin provided (\'https://www.youtube.com\') does not ' +
-      'match the recipient window\'s origin (\'http://localhost:9001\').'
+      // Triple backslashes are needed because backslashes are escape characters
+      // in both regexes and strings: https://stackoverflow.com/a/5514380
+      'The target origin provided \\\(\'https://www\.youtube\.com\'\\\) does ' +
```

Comment:
IIRC this error only occurs in certain machines -- it was recently added to the "error to ignore" list because of this reason. Can you please test this change in a machine where this console error needs to be ignored jsut to be sure? Perhaps @srijanreddy98 can help (his machine used to throw this error).

Reply:
Ah, I have tested it -- it arose once in the Actions tests, I haven't seen it locally. I think it might depend on what YouTube is serving at the time the test is run. I tested it in the console.log with the actual message from the test log to ensure that the match happens correctly.